src/azure-cli/azure/cli/command_modules/find/commands.py

```diff
@@ -6,5 +6,5 @@
 
 def load_command_table(self, _):
     with self.command_group('') as g:
-        g.custom_command('find', 'process_query', is_preview=True)
+        g.custom_command('find', 'process_query', is_preview=False)
```

Comment:
If you want to remove Preview flag, you should update history.rst to mark `az find` in GA.